.changeset/wet-mice-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@lens-protocol/wagmi": major
+---
+
+upgrade wagmi bindings to wagmi v1
```

Comment:
```suggestion
**Upgraded** wagmi to v1
```

Reply:
just for consistency as i see we recently got out of sync with changeset namings